test/cpp/tensorexpr/tests.h

```diff
@@ -231,7 +231,14 @@ namespace jit {
   _(Kernel_4)                               \
   _(FuserPass_1)                            \
   _(FuserPass_2)                            \
-  _(TrainBasic)
+  _(TrainBasic)                             \
```

Comment:
Can you move these up to be with the other LoopNest tests (so just above Kernel_1